cmd/delete/account/account.go

```diff
@@ -0,0 +1,58 @@
+package account
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+
+	"github.com/loginradius/lr-cli/cmdutil"
+
+	"github.com/spf13/cobra"
+)
+
+var fileName string
+
+var inpEmail string
+
+func NewaccountCmd() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     "account",
+		Short:   "delete account",
+		Long:    `This commmand deletes account`,
+		Example: heredoc.Doc(`$ lr delete account --email <email>`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if inpEmail == "" {
+				return &cmdutil.FlagError{Err: errors.New("`email` is required argument")}
+			}
+
+			return delete(inpEmail)
+
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringVarP(&inpEmail, "email", "e", "", "emailID")
+
+	return cmd
+}
+
+func delete(Email string) error {
+	res1, err := cmdutil.GetAPICreds()
+	var emailstring = Email
+	url := "https://api.loginradius.com/identity/v2/manage/account?apikey=" + res1.Key + "&apisecret=" + res1.Secret + emailstring
+	req, _ := http.NewRequest("DELETE", url, nil)
+	req.Header.Add("content-Type", "application/x-www-form-urlencoded")
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		panic(err)
+	}
+	defer res.Body.Close()
```

Comment:
Here we can use the admin console API, instead of login radius API. Check customer-management microservice of admin console.